# Comments provide naming from Technical Customer Information

# LocAtr_FoVRange
float32[99] fov
# LocAtr_AziAngArr
float32[99] azimuth
# LocAtr_RangScaEle
float32[11] elevation_range_scaling
# LocAtr_EleAngArr
float32[11] elevation
